src/components/SQForm/SQFormDateTimePicker.tsx

```diff
@@ -5,8 +5,8 @@ import TextField from '@material-ui/core/TextField';
 import {DateTimePicker, BaseDateTimePickerProps} from '@material-ui/pickers';
 import {ClickAwayListener, makeStyles} from '@material-ui/core';
 import {useForm} from './useForm';
-import BaseFieldProps from '../../types/BaseFieldProps';
-import {ParsableDate} from '@material-ui/pickers/constants/prop-types';
+import type BaseFieldProps from '../../types/BaseFieldProps';
```

Comment:
[question] don't we need to import this as if it's a named export (`{ BaseFieldProps }`) instead of default?

Reply:
@lucas-homer Thanks for the feedback!  BaseFieldProps actually is the default export from the imported file.

Reply:
I think Lucas was mentioning that we should change this import to import from the `types` index file instead of the `BaseFieldProps` file itself.

Reply:
Ah. Thanks for clarifying @20BBrown14.